graylog2-server/src/main/java/org/graylog2/indexer/fieldtypes/IndexFieldTypePollerPeriodical.java

```diff
@@ -164,20 +168,30 @@ private void poll(Collection<IndexSetConfig> indexSetConfigs) {
         indexSetConfigs.stream()
                 .filter(config -> !config.fieldTypeRefreshInterval().equals(Duration.ZERO))
                 .filter(IndexSetConfig::isWritable)
-                .filter(config -> {
+                .forEach(config -> {
                     final Instant previousPoll = lastPoll.getOrDefault(config.id(), Instant.MIN);
                     final Instant nextPoll = previousPoll.plusSeconds(
                             config.fieldTypeRefreshInterval().getStandardSeconds());
-                    return !Instant.now().isBefore(nextPoll);
-                })
-                .forEach(this::poll);
+                    if (!Instant.now().isBefore(nextPoll)) {
```

Comment:
Won't `Instant.now().isAfter()` be slightly more readable?

Reply:
I had the same thought. I think the intention was that it should also start when when `now == nextPoll`.